bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -535,6 +535,7 @@ spec:
             --nbctl-daemon-mode \
             --nb-cert-common-name "{{.OVN_CERT_CN}}" \
             --enable-multicast
+            --enable-egress-ip
```

Comment:
can you set this in `bindata/network/ovn-kubernetes/004-config.yaml` instead?

Reply:
Yes, this is what I first tried doing, however I didn't understand the sections (https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/ovn-kubernetes/004-config.yaml#L20) and how they're specified. I tried several variants of `OVNKubernetesFeatureConfig`, but it continued returning errors. Any pointer?

Reply:
It's based on the name of the field in the top-level config object, not the name of the _type_ of the field. So it should be `[ovnkubernetesfeature]`

Reply:
Got it, thanks! Will update the PR